bootstrap.bat

```diff
@@ -1,4 +1,7 @@
-nuget restore
+@echo off
+
+msbuild /t:restore

```

Comment:
Fair. But path parsing order, multiple scattered versions, and explicit binaries necessary to get a solution in working order mimic the package system, so I think the addition of this tool to be prudent. That being said, I don't mind either way.